support/tech-resources/deep-links-self-serve.md

```diff
@@ -5,152 +5,229 @@ description: "A guide to including iOS universal links and Android App Links in
 
 ## Introduction
 
-Both Universal and App links are a mechanism in iOS (9 and later) and Android (6.0 and later) that allows you to link deeply into your apps from web pages and HTML email. ***Note: Deep Link, Universal Link, and App Link terms will be used interchangeably.*** To use deep linking, you must publish the URLs you'd like your app to handle and then add support for them into the app itself. When your users receive an email containing a deep link, the mobile device then recognizes the domain and path of the URL and triggers your app. As a result, it's important to keep the URLs in your HTML emails intact to ensure your deep links function correctly.
+Email is often read and engaged with on mobile devices. Both iOS (9 and later) and Android (6.0 and later) provide a way for links in your content to take your users directly into an app that you have created, providing your users with a richer interactive experience.
```

Comment:
add: The links that connect a user directly to your app are referred to as deep links.

Reply:
Done.